src/containers/EditListingPage/EditListingWizard/EditListingWizardTab.js

```diff
@@ -163,6 +164,7 @@ const EditListingWizardTab = props => {
       listing,
       panelUpdated: updatedTab === tab,
       params,
+      listingACCs,
```

Comment:
Should pass **listingACCs** direct to the **EditListingDetailsPanel** instead of all order panels.